packages/plugin-hardhat/src/verify-proxy.ts

```diff
@@ -191,12 +218,14 @@ async function fullVerifyTransparentOrUUPS(
     if (!isEmptySlot(adminAddress)) {
       console.log(`Verifying proxy admin: ${adminAddress}`);
       try {
-        await verifyContractWithCreationEvent(
+        await verifyWithArtifactOrFallback(
           hre,
+          hardhatVerify,
           etherscanApi,
           adminAddress,
           [verifiableContracts.proxyAdmin],
-          errors,
+          errorReport,
+          false,
```

Comment:
Can you explain why we're treating this one differently in this way?

Reply:
Added reasoning in comments.